### ISSUE TITLE
Syncing duplicates Accounts

### ISSUE THREAD

**Issue by simonszu** [saenzramiro/rambox#1311]
### Steps to reproduce

1. Create a bunch of services on Computer A
2. Sync it to "The Cloud" with "Synchronize -> Backup"
3. Go to Computer B, and "Synchronize -> Restore Configuration"
4. Go to Computer A, add another service, sync it with "Backup"
5. Go to Computer B, and try to get the new service added. "Sync -> Check for updated Backup" yields "A new backup is available"
6. Use "Synchronize -> Restore Configuration" on Computer B. 

### Expected behavior

Expected: The new service added on Computer A appears on Computer B, and nothing more. Or, even more, a "Check for updated backup" already applies the delta between the current configuration and the configuration in the cloud. 

### Actual behavior

All Services in the Backup are restored a second time, so i have two WhatsApps, two Steams, etc., but only one instance of the new Service. What makes it worse is that you cannot see what services are already logged in on the main tab, so you only can remove the not logged-in duplicates by trial-and-error. 

### ENVIRONMENT

> Rambox: 0.5.13

> OS: Windows 10 on Computer B, Arch Linux on Computer A

> Arch: x64

**Comment by simonszu** [comment]
This is still an issue with Rambox 0.5.14. How am i expected to use the sync feature to make this work? "Backup configuration" on one computer, and then "Restore configuration" on another? Why is there a "Check for updated backup" button anyways?

**Comment by simonszu** [comment]
New post for a new experiment. Now playing the sync workflow on two machines with an eye on the JSON payload. 

I have a logged in Rambox instance on the first computer. There are four services anabled. The JSON synced to the API contained all four services and nothing more. 

On the second computer i have a clean Rambox instance and initiated a login. I suppose the sync happens before Rambox practically restarts and shows the intro animation with the flying icons, since i cannot find the JSON response by the API with the enabled services, although they are now enabled. I add a 5th service. After hitting sync, i see a JSON with 5 enabled services being sent to the API.

Back to computer 1. I will initiate a download of the settings (restore configuration). The only entry i see in the developer console is the `delegation` with the request payload. The API returns HTTP 200, but nothing more happens. A click on "check for updated settings" causes the notification to pop up, but there is no new entry in the network log. Does it even call the API? After this, "restore configuration" still only causes a delegation entry, but no "real" API hit/response. 

I restart Rambox on Computer 1. After that, a click on "check for updated config" causes a delegation hit. A "restore" causes only a delegation hit as well, but no real download. An "upload config" causes a proper API hit, which is viewable in the network log. _Only after that_ a "download config" results in an API response. Sadly, rambox restarts right after that, so i cannot capture the JSON with the payload, but after Rambox's restart, the services which were there beforehand are duplicated. The new service added on computer 2 isn't there (no surprise, since the last upload from computer 1 didn't include the service). 

So i think that this is not only an issue about adding services a second time after sync download, but also an issue about the API not working properly.

**Comment by ArtursO** [comment]
Yeah this is quite annoying and pretty much renders syncing a configuration unusable, especially if you have a bunch of services and have to go into rambox while it tries to double load a bunch of services and disabling each one by one.

**Comment by simonszu** [comment]
This issue is still happening. Great move, automatically trying to close a year old issue with no reaction by a developer, and the "wontfix" label. I don't think I'll buy a pro license now.

**Comment by MurzNN** [comment]
Confirm that issue still exists on 0.6.2 version - after each restoring from backup I got duplicates of tabs.

**Comment by simonszu** [comment]
No, the issue still persists.

**Comment by simonszu** [comment]
No, the issue still persists.

**Comment by simonszu** [comment]
No, the issue still persists.

**Comment by MurzNN** [comment]
bump! :angry:

**Comment by simonszu** [comment]
No the issue still persists, as well as serious management issues in this repository. Ignoring an issue for over 18 months, not even a "acknowledged, but no time, feel free to create a PR".

Maybe I'll look into hamsket now. They do not have a sync as well, but they communicate it openly.

Of course, I will re-open this issue every two months. Let's see when the maintainers will look into it.

**Comment by stefanuddenberg** [comment]
This issue makes using Rambox much more difficult than it should be.

**Comment by simonszu** [comment]
The issue is still persisting, by the way.

**Comment by simonszu** [comment]
The issue is still persisting, by the way

**Comment by simonszu** [comment]
Still an issue :)

**Comment by simonszu** [comment]
Hi, 
this is your irregular but persistent reminder that this issue is still persisting. Really looking forward to this issue's 4-year anniversary 🥳

**Comment by simonszu** [comment]
OK, tomorrow is the 4th birthday of this issue, so far no acknowledge, nothing, only a bunch of unsatisfied users. I have no idea how this mindset enables anyone to make money with this software. 

I for myself will switch to https://github.com/TheGoddessInari/hamsket which is a fork of rambox. It has everything rambox provides and more (e.g. acknowledging of issues on GitHub). Even the sync issue is fixed, they just have a manual export and import of settings. 

Of course, i'll let this issue stay open, since it still persists.

**Comment by MurzNN** [comment]
I switched to https://github.com/getferdi/Ferdi that provides same functionality, like Rambox, but development is much more active, take a look on it

**Comment by simonszu** [comment]
Yeah, i do it infrequently. But last time i checked they hat not so good support for own apps, at least for their sync server (which you can selfhost). And my work chat is a not so well known service
